File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/errorresponsemessagepaymentrequired.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import beezup_common_usererrormessage as shared_beezup_common_usererrormessage
from ..shared import errorresponsemessagepaymentrequiredlinks as shared_errorresponsemessagepaymentrequiredlinks


@dataclass_json
@dataclasses.dataclass
class ErrorResponseMessagePaymentRequired:
    errors: list[shared_beezup_common_usererrormessage.BeezUpCommonUserErrorMessage] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('errors') }})
    links: shared_errorresponsemessagepaymentrequiredlinks.ErrorResponseMessagePaymentRequiredLinks = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('links') }})
    

File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/errorresponsemessagepaymentrequired.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { BeezUpCommonUserErrorMessage } from "./beezupcommonusererrormessage";
import { ErrorResponseMessagePaymentRequiredLinks } from "./errorresponsemessagepaymentrequiredlinks";



export class ErrorResponseMessagePaymentRequired extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=errors", elemType: BeezUpCommonUserErrorMessage })
  errors: BeezUpCommonUserErrorMessage[];

  @SpeakeasyMetadata({ data: "json, name=links" })
  links: ErrorResponseMessagePaymentRequiredLinks;
}
